src/browser/tab/mod.rs

```diff
@@ -382,31 +400,31 @@ impl<'a> Tab {
         sleep(Duration::from_millis(scaled_millis));
     }
 
-    pub fn wait_for_element(&self, selector: &str) -> Fallible<Element<'_>> {
+    pub fn wait_for_element(&self, selector: Selector) -> Fallible<Element<'_>> {
```

Comment:
It'd be nice to not have to break backwards compatibility here. Could you add a separate method for xpath, [like how Puppeteer has a separate method](https://pptr.dev/#?product=Puppeteer&version=v8.0.0&show=api-pagewaitforxpathxpath-options)?

Reply:
Or use `Into` and make the default behavour for `"#asdfad".into()` be that it becomes a CSS selector?

Reply:
Originally i had a seperate method,but then i decided to follow what fantoccini did and made a function that accepted a enum, i will make a seperate method then,

Reply:
> Or use `Into` and make the default behavour for `"#asdfad".into()` be that it becomes a CSS selector?

this seems like a nice idea,  i will try doing that

Reply:
cool, and if you call `into()` inside `find_element` then people can still call `find_element('#example')` (i.e. it won't change the public API)

Reply:
> cool, and if you call `into()` inside `find_element` then people can still call `find_element('#example')` (i.e. it won't change the public API)

Got it